packages/html-spec/src/listview/listview.json

```diff
@@ -0,0 +1,98 @@
+{
+    "$schema": "../../schemas/component.schema.json",
+    "title": "Kendo Listview HTML Spec",
+    "component": "listview",
+    "selector": [
+        ".k-listview"
+    ],
+    "states": [
+        {
+            "name": "disabled",
+            "selector": [
+                ".k-disabled"
+            ]
+        },
+        {
+            "name": "focus",
+            "selector": [
+                ":focus",
+                ".k-focus"
+            ]
+        }
```

Comment:
@veselints do we need the entire ListView to be focusable? According to [the current spec](https://github.com/telerik/web-components-ux/blob/master/docs/components/listview/keyboard-navigation.md) - no. If this is indeed the case, we should remove this from the html spec.

Reply:
still waiting?

Reply:
it shouldn't be focusable.